parser/src/newparser/syntax_notes.md

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
I think this file should be removed and its contents can go partly into issues and partly into the [WIP Spec](https://github.com/ethereum/fe/blob/master/spec/index.md)

We could also just introduce an undecided/open questions section in the spec to host some of this.

Reply:
Oh yeah, this was a note-to-self kinda doc. I'll remove and update the spec/issues later.